### DISCUSSION TITLE
After taming a goat the program crashed

### DISCUSSION THREAD

**Discussion by gnezamay** [Cediner/ArdClient#94] [Q&A]
Hi!

After taming a goat the program crashed
![59](https://github.com/Cediner/ArdClient/assets/130748454/ae353bb6-8a44-45fe-9004-55fb3c3bbbf1)

also the window for studying the animal does not open

**Comment by Cediner** [comment]
Is the problem still relevant?

**Reply by gnezamay** [reply]
After taming 2 wild goats, the client did not collapse, so it is probably irrelevant.
Inspection does not open, but this is not critical.
![изображение](https://github.com/Cediner/ArdClient/assets/130748454/cdd03488-e3c0-43db-9f24-ea1d529770df)

Thank you for your attention
